core/block.ts

```diff
@@ -1641,6 +1641,22 @@ export class Block implements IASTNodeLocation, IDeletable {
       );
     }
 
+    // Validate that each arg has a corresponding message
+    let n = 0;
+    while (json['args' + n]) {
+      if (json['message' + n] === undefined) {
+        throw Error(
+          warningPrefix +
+            'args' +
+            n +
+            ' must have a corresponding message (message' +
+            n +
+            ').',
```

Comment:
nit: Could you switch this to a [template string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) so that it's a bit easier to read?